orga/app/components/routes/register-form.js

```diff
@@ -58,6 +66,9 @@ export default class RegisterForm extends Component {
 
   @action
   async register() {
+    if (this.isFormNotValid) {
+      return this.set('isLoading', false);
```

Comment:
Octane : this.isLoading = false

Reply:
Oui mais Orga n'est pas encore totalement sur Octane. Actuellement, tous les isLoading sont gérer avec des set()